test/setup.ts

```diff
@@ -13,6 +13,7 @@ export default () => {
   process.env.PAYMENT_FEE = "100";
   process.env.LANDING_PAGE_URL = "/register-an-overseas-entity/sold-land-filter?start=0";
   process.env.UPDATE_LANDING_PAGE_URL = "/update-an-overseas-entity/overseas-entity-query";
+  process.env.UPDATE_CONFIRMATION_LANDING_PAGE_URL = "/update-an-overseas-entity-confirmation";
```

Comment:
this isn't required I don't think, shouldn't be any new env variables being added UPDATE_CONFIRMATION_LANDING_PAGE_URL

Reply:
Done